.travis.yml

```
@@ -4,20 +4,33 @@ compiler:
   - gcc
   - clang
 
-sudo: required
+env:
+  global:
+    - PACKAGES="libc6-dev uuid-dev libpam0g-dev libcurl4-openssl-dev dh-autoreconf autotools-dev clang cmake pkg-config texinfo libssl-dev libsubunit-dev libsubunit0 checkinstall"
+
+before_install:
+  - sudo -H apt-get -q -qq -y update > /dev/null
+  - sudo -H apt-get -q -qq -y install $PACKAGES > /dev/null
 
 install:
-  - sudo -H apt-get -q -qq -y update
-  - sudo -H apt-get -q -qq -y install libc6-dev uuid-dev libpam0g-dev libcurl4-openssl-dev dh-autoreconf autotools-dev clang cmake pkg-config texinfo libssl-dev libsubunit-dev libsubunit0
-  - sudo -H ./ci/dependencies.sh
+  - sudo -H ci/dependencies.sh > /dev/null
+
+stages:
+  - name: build
+  - name: package
```

I'd recommend putting back in the `test` stage since Travis is gonna run those. You don't have to define the jobs just update this to:

```suggestion
  - name: build
  - name: test
  - name: package
```

---

order does not matter here